src/components/ec-radio-btn-group/ec-radio-btn-group.spec.ts

```diff
@@ -0,0 +1,209 @@
+import type { MountingOptions } from '@vue/test-utils';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { mount } from '@vue/test-utils';
+
+import type { CVueWrapper } from '../../../tests/utils/global';
+import { EcRadioBtn, EcRadioBtnGroup } from '../../main';
+import type { RadioButtonGroupProps } from './types';
+
+function mountRadioBtnGroup(props?: Partial<RadioButtonGroupProps>, mountOpts?: MountingOptions<RadioButtonGroupProps>) {
+  return mount(
+    EcRadioBtnGroup as any, // eslint-disable-line
+    {
+      props: {
+        options: [
+          { value: 'y', label: 'Yes' },
+          { value: 'n', label: 'No' },
+        ],
+        ...props,
+      },
+      ...mountOpts,
+    },
+  ) as CVueWrapper;
+}
+
+describe('EcRadioBtn', () => {
+  it('should render properly', () => {
+    const wrapper = mountRadioBtnGroup();
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  describe('props', () => {
+    it('should render the child component (radio-btn) with all the given props', () => {
+      const wrapper = mountRadioBtnGroup({
+        options: [{ value: 'y', label: 'Yes', description: 'Confirm' }, { value: 'n', label: 'No', description: 'Reject' }],
+        isDisabled: true,
+        isTextInline: true,
+        modelValue: 'y',
+        errorMessage: 'A testing error message',
+      });
+      const givenPropsRadio1 = wrapper.findAllComponents(EcRadioBtn)[0].vm;
+      const givenPropsRadio2 = wrapper.findAllComponents(EcRadioBtn)[1].vm;
+
+      const expectedPropsRadio1 = {
+        isDisabled: true,
+        isTextInline: true,
+        modelValue: 'y',
+        label: 'Yes',
+        description: 'Confirm',
+        hasError: true,
+        value: 'y',
+      };
+      const expectedPropsRadio2 = {
+        isDisabled: true,
+        isTextInline: true,
+        modelValue: 'y',
+        label: 'No',
+        description: 'Reject',
+        hasError: true,
+        value: 'n',
+      };
+
+      expect(givenPropsRadio1.isDisabled).toBe(expectedPropsRadio1.isDisabled);
+      expect(givenPropsRadio1.isTextInline).toBe(expectedPropsRadio1.isTextInline);
+      expect(givenPropsRadio1.label).toBe(expectedPropsRadio1.label);
+      expect(givenPropsRadio1.description).toBe(expectedPropsRadio1.description);
+      expect(givenPropsRadio1.hasError).toBe(expectedPropsRadio1.hasError);
+      expect(givenPropsRadio1.value).toBe(expectedPropsRadio1.value);
+
+      expect(givenPropsRadio2.isDisabled).toBe(expectedPropsRadio2.isDisabled);
+      expect(givenPropsRadio2.isTextInline).toBe(expectedPropsRadio2.isTextInline);
+      expect(givenPropsRadio2.label).toBe(expectedPropsRadio2.label);
+      expect(givenPropsRadio2.description).toBe(expectedPropsRadio2.description);
+      expect(givenPropsRadio2.hasError).toBe(expectedPropsRadio2.hasError);
+      expect(givenPropsRadio2.value).toBe(expectedPropsRadio2.value);
```

Comment:
Can we use here `$props` and `toEqual(expectedPropsRadio1)`?

Reply:
so i can do this 
` 
const expectedPropsRadio1 = wrapper.findAllComponents(EcRadioBtn)[0].props() 
expect(givenPropsRadio1).toEqual(expectedPropsRadio1);
`

but i get this warning 

`expect(givenPropsRadio1).toEqual(expectedPropsRadio1);`